pkgs/development/libraries/proj/default.nix

```diff
@@ -44,6 +36,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DUSE_EXTERNAL_GTEST=ON"
+    "-DRUN_NETWORK_DEPENDENT_TESTS=OFF"
   ];
 
   doCheck = stdenv.is64bit;
```

Comment:
Is this still true?

Reply:
I wouldn't know.

Reply:
`pkgsi686Linux.proj` builds just fine with tests enabled, so I set `doCheck = true`.